python/KMS_KEYS_TO_NOT_DELETE.py

```diff
@@ -0,0 +1,452 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+'''
+#####################################
+##            Gherkin              ##
+#####################################
+
+Rule Name:
+	KMS_KEYS_TO_NOT_DELETE
+
+Description:
+	Check that Customer Managed keys are not scheduled for deletion. The rule is NON_COMPLAINT if the Customer Managed Keys are scheduled for deletion. This rule does not include AWS-managed and Imported Keys.
+
+Trigger:
+	Periodic
+
+Reports on:
+	AWS::KMS::Key
+
+Rule Parameters:
+	kmsKeyIds (Optional)
+	Comma-separated list of specific Customer Managed Key Ids, that are expected not to be scheduled for deletion.
+
+Scenarios:
+	Scenario 1:
+	Given: No CMKs in the Account
```

Comment:
`in the Account` or `in the AWS Account` is not accurate. Config is regional. Remove any mention of it.